### ISSUE TITLE
Fix extension activation

### ISSUE THREAD

**Issue by dcrouch79** [angular/vscode-ng-language-service#1198]
Version 11.2.6 of extension fails to activate.

Version 11.2.5 working ok.

**Environment**
- Windows 10 + WLS2
- VS Code 1.54.1
- Angular 10

**Logs**
<details>
  <summary>VS Code Console Logs</summary>

```
Activating extension 'Angular.ng-template' failed: Unexpected token '?'.
_logMessageInConsole @ abstractExtensionService.ts:718
_logOrShowMessage @ abstractExtensionService.ts:732
$onExtensionActivationError @ mainThreadExtensionService.ts:64
_doInvokeHandler @ rpcProtocol.ts:417
_invokeHandler @ rpcProtocol.ts:402
_receiveRequest @ rpcProtocol.ts:318
_receiveOneMessage @ rpcProtocol.ts:245
(anonymous) @ rpcProtocol.ts:110
fire @ event.ts:622
fire @ ipc.net.ts:468
_receiveMessage @ ipc.net.ts:821
(anonymous) @ ipc.net.ts:660
fire @ event.ts:622
acceptChunk @ ipc.net.ts:241
(anonymous) @ ipc.net.ts:202
(anonymous) @ browserSocketFactory.ts:178
fire @ event.ts:622
_fileReader.onload @ browserSocketFactory.ts:62
load (async)
y @ browserSocketFactory.ts:58
create @ browserSocketFactory.ts:162
connect @ browserSocketFactory.ts:211
o @ remoteAgentConnection.ts:192
n @ remoteAgentConnection.ts:233
i @ remoteAgentConnection.ts:310
h @ remoteAgentConnection.ts:352
E @ remoteAgentConnection.ts:421
async function (async)
E @ remoteAgentConnection.ts:420
(anonymous) @ remoteExtensionHost.ts:128
Promise.then (async)
start @ remoteExtensionHost.ts:104
nr @ extensionHostManager.ts:67
_createInstance @ instantiationService.ts:110
createInstance @ instantiationService.ts:76
(anonymous) @ abstractExtensionService.ts:447
_startExtensionHosts @ abstractExtensionService.ts:446
_initialize @ abstractExtensionService.ts:411
(anonymous) @ extensionService.ts:102
requestIdleCallback (async)
e.runWhenIdle @ async.ts:823
(anonymous) @ extensionService.ts:101
Promise.then (async)
x @ extensionService.ts:99
_createInstance @ instantiationService.ts:110
_createServiceInstance @ instantiationService.ts:230
_createServiceInstanceWithOwner @ instantiationService.ts:219
_createAndCacheServiceInstance @ instantiationService.ts:208
_safeCreateAndCacheServiceInstance @ instantiationService.ts:151
_getOrCreateServiceInstance @ instantiationService.ts:135
get @ instantiationService.ts:54
initLayout @ layout.ts:255
(anonymous) @ workbench.ts:137
invokeFunction @ instantiationService.ts:61
startup @ workbench.ts:130
open @ desktop.main.ts:123
async function (async)
open @ desktop.main.ts:111
re @ desktop.main.ts:364
m.load.removeDeveloperKeybindingsAfterLoad @ workbench.js:32
(anonymous) @ bootstrap-window.js:166
async function (async)
(anonymous) @ bootstrap-window.js:162
r._invokeFactory @ loader.js:1114
r.complete @ loader.js:1124
r._onModuleComplete @ loader.js:1750
r._onModuleComplete @ loader.js:1762
r._resolve @ loader.js:1710
r.defineModule @ loader.js:1353
p @ loader.js:1800
g @ loader.js:841
(anonymous) @ tunnelServiceImpl.ts:15
(anonymous) @ fake:1
r._createAndEvalScript @ loader.js:844
(anonymous) @ loader.js:826
v @ loader.js:945
(anonymous) @ loader.js:950
readFileAfterClose @ internal/fs/read_file_context.js:63
```

</details>

**Comment by vitale232** [comment]
I'm seeing something similar in my Win 10 + WSL2 (Ubuntu 20.04) setup using VS Code 1.54.1

When upgrading to v11.2.6 of the VS Code extension, the language service fails to load in my remote VS Code session.

If I run the `Angular: Restart Angular Language server` command from the command palette, I get a notification with the following error message:

`Command 'Angular: Restart Angular Language server' resulted in an error (command 'angular.restartNgServer' not found)`

**Comment by kyliau** [comment]
We've rolled back some changes in v11.2.6, and published [v11.2.7](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.2.7) while we investigate this issue.
